presto-hive/src/main/java/com/facebook/presto/hive/metastore/thrift/ThriftMetastoreUtil.java

```diff
@@ -331,7 +331,7 @@ public static HiveColumnStatistics fromMetastoreApiColumnStatistics(ColumnStatis
             LongColumnStatsData longStatsData = columnStatistics.getStatsData().getLongStats();
             OptionalLong min = longStatsData.isSetLowValue() ? OptionalLong.of(longStatsData.getLowValue()) : OptionalLong.empty();
             OptionalLong max = longStatsData.isSetHighValue() ? OptionalLong.of(longStatsData.getHighValue()) : OptionalLong.empty();
-            OptionalLong nullsCount = longStatsData.isSetNumNulls() ? OptionalLong.of(longStatsData.getNumNulls()) : OptionalLong.empty();
+            OptionalLong nullsCount = longStatsData.isSetNumNulls() ? fromMetastoreNullsCount(longStatsData.getNumNulls()) : OptionalLong.empty();
```

Comment:
There are other places below that need an update (search for "nullsCount")

Reply:
@findepi , thanks for your correction. I have add the same converstion for other nullsCounts in method [fromMetastoreApiColumnStatistics](https://github.com/prestodb/presto/blob/0e4290ce65073eab649d21f15f4882036e88eb25/presto-hive/src/main/java/com/facebook/presto/hive/metastore/thrift/ThriftMetastoreUtil.java#L328) 
I thinks other methods like [createDateStatistics](https://github.com/prestodb/presto/blob/0e4290ce65073eab649d21f15f4882036e88eb25/presto-hive/src/main/java/com/facebook/presto/hive/metastore/thrift/ThriftMetastoreUtil.java#L653) which is used to write back the statistics information to Hive.
What your opinion?

Reply:
When _writing_, -1 should by no means be a written value, so no need to change there.